lcd/lcd_api.py

```diff
@@ -151,6 +152,21 @@ def putstr(self, string):
         for char in string:
             self.putchar(char)
 
+    def custom_char(self, location, charmap):
+        """Write a character to one of the 8 CGRAM locations"""
+        location &= 0x7
+        self.hal_write_command(self.LCD_CGRAM | (location << 3))
+        time.sleep_us(40)
+        for i in range(8):
+            self.hal_write_data(charmap[i])
+            time.sleep_us(40)
+        self.move_to(self.cursor_x, self.cursor_y)
+
+    def putcustom(self, location):
+        """Writes a custom character to the LCD"""
+        location &= 0x7
+        self.putchar(chr(location))
+
```

Comment:
lcd.putchar(chr(location)) does the same thing as lcd.putcustom(location)

Do you think we still need putcustom ?